### PR TITLE
Display the role of the governing body in secretaries ( "Діловодства" )tab  

### PR DESCRIPTION
**Pull request by christykobyletska**
… )tab

dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/pages/userPage/Secretaries/UserGoverningBodySecretaryTable.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+interface props {
+    UserId: string;
+}
+
+export const UserGoverningBodySecretaryTable = ({UserId}: props) => {
+    const [isLoadingActive, setIsLoadingActive] = useState<boolean>(true);
+    const [isLoadingPrev, setIsLoadingPrev] = useState<boolean>(true);
+
+    const [data, setData] = useState<SecretaryModel[]>();
```
Comment (sharix4) [inline src/pages/userPage/Secretaries/UserGoverningBodySecretaryTable.tsx:19]: data -should be called GoverningBodyAdmins

File: src/pages/userPage/Secretaries/UserGoverningBodySecretaryTable.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    const [prevData, setPrevData] = useState<SecretaryModel[]>();
+
+    const fetchData = async () => {
+        setIsLoadingActive(true);
+        try {
+            await getUsersAdministrations(UserId).then(response => {
+                setData(response.data);
```
Comment (sharix4) [inline src/pages/userPage/Secretaries/UserGoverningBodySecretaryTable.tsx:26]: setDataAdministrations

File: src/pages/userPage/Secretaries/UserGoverningBodySecretaryTable.tsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        } catch (error) {
+            showError(error.message);
+        } finally {
+            setIsLoadingActive(false);
+        }
+
+        setIsLoadingPrev(true);
+        try {
+            await getUsersPreviousAdministrations(UserId).then(resp => {
+                setPrevData(resp.data)
```
Comment (sharix4) [inline src/pages/userPage/Secretaries/UserGoverningBodySecretaryTable.tsx:37]: setPrevDataAdministrations

**Review by sharix4 — CHANGES_REQUESTED** [review]

File: src/pages/userPage/Secretaries/UserGoverningBodySecretaryTable.tsx (hunk 3 above)
Comment (christykobyletska) [reply src/pages/userPage/Secretaries/UserGoverningBodySecretaryTable.tsx]: done

File: src/pages/userPage/Secretaries/UserGoverningBodySecretaryTable.tsx (hunk 1 above)
Comment (christykobyletska) [reply src/pages/userPage/Secretaries/UserGoverningBodySecretaryTable.tsx]: done

File: src/pages/userPage/Secretaries/UserGoverningBodySecretaryTable.tsx (hunk 2 above)
Comment (christykobyletska) [reply src/pages/userPage/Secretaries/UserGoverningBodySecretaryTable.tsx]: done

File: src/pages/userPage/Secretaries/UserGoverningBodySecretaryTable.tsx (hunk 3 above)
Comment (sharix4) [reply src/pages/userPage/Secretaries/UserGoverningBodySecretaryTable.tsx]: excellent

File: src/pages/userPage/Secretaries/UserGoverningBodySecretaryTable.tsx (hunk 2 above)
Comment (sharix4) [reply src/pages/userPage/Secretaries/UserGoverningBodySecretaryTable.tsx]: excellent

**Review by sharix4 — APPROVED** [review]

**Review by OleksandrBilan — APPROVED** [review]

**Review by nicoolle — APPROVED** [review]

**Review by Vinnytile — APPROVED** [review]

**Pull request merged**